Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:10
+
+RUN mkdir -p /usr/src
+WORKDIR /usr/src/
+
+COPY ./ /usr/src
+RUN chown -R node:node .
+
+USER node
+
+RUN yarn install
```

Comment:
You can make the Docker build much faster by installing your dependencies before copying the source code across. Something like
```docker
WORKDIR /usr/src

RUN chown -R node:node /usr/src
USER node

ADD ./package.json /usr/src
ADD ./yarn.lock /usr/src
RUN yarn install

COPY ./ /usr/src
```

That will speed up your build by [making use of Docker's cached layers](http://bitjudo.com/blog/2014/03/13/building-efficient-dockerfiles-node-dot-js/).